modules/multi-architecture-enabling-64k-pages.adoc

```diff
@@ -0,0 +1,55 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/multi-architecture-configuration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="multi-architecture-enabling-64k-pages_{context}"]
+
+= Enabling 64k pages on the {op-system-first} kernel 
+
+You can enable 64k memory pages in the {op-system-first} kernel on the 64-bit ARM compute machines in your cluster. The 64k page kernel specification can be used for large GPU or high memory workloads. This is done using the Machine Config Operator (MCO) which updates the {op-system} kernel. You can create a custom machine config pool to add 64k pages to your nodes. 
```

Comment:
> 64k memory pages

I wonder if we should call out the feature as `64k pagesize kernel` throughout the doc. Although it's shorter and clear enough to call it out as "64k pages", I'd keep some more formality/precision in the documentation.

Reply:
I've been using `64k pages` as the main feature name, but in introductory sentences or in the beginning of some procedures I've been using things like `64k page kernel` or `64k memory pages` just to add more details when introducing the topic.

Reply:
Should we use `64k page size kernel` as main feature name based on the discussion in the [other PR](https://github.com/openshift/openshift-docs/pull/69875#discussion_r1447719960) and for consistency with the RHEL doc?